src/lv_misc/lv_area.h

```diff
@@ -28,6 +28,10 @@ extern "C" {
 #define LV_COORD_MAX (16383) /*To avoid overflow don't let the max [-32,32k] range */
 #define LV_COORD_MIN (-16384)
 
+#define LV_EXT_CLICK_AREA_OFF   0
+#define LV_EXT_CLICK_AREA_TINY  1
+#define LV_EXT_CLICK_AREA_FULL  2
```

Comment:
If `lv_area_ext_is_point_on` is not required (see below) they are also not required.